### ISSUE TITLE
How to build for Android?

### ISSUE THREAD

**Issue by Alireza-Ta** [MasterQ32/SDL.zig#144]
Hi there!

Is there a way to build for Android based on `android-project` directory in the original SDL repository?

Thanks!

**Comment by MasterQ32** [comment]
No, but you can probably build that version yourself, and link against it instead of the stubs/vendored ones
